web/src/app/converters/submission-data-converter.ts

```diff
@@ -107,7 +107,15 @@ function taskDataPbToModelValue(
   } else if (drawGeometryResult)
     return geometryPbToModel(drawGeometryResult.geometry!) as Polygon;
   else if (captureLocationResult)
-    return new Point(coordinatesPbToModel(captureLocationResult.coordinates!));
+    return new Point(
+      coordinatesPbToModel(captureLocationResult.coordinates!),
+      captureLocationResult.accuracy === null
```

Comment:
Does the elvis operator help here? `captureLocationResult.accuracy ?: null`?

Reply:
see https://github.com/google/ground-platform/pull/2102